cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -447,7 +447,17 @@ values for Cloudera may vary from the above appendix:
     :start-after: .. _cloudera-starting-services-java-heapmax:
     :end-before: .. end_of_list
 
-At this point, the CDAP installation is configured and is ready to be installed. Review
+**The CDAP system user:** As Hadoop resolves users at the NameNode, the ``cdap`` user must
+be added there, or name resolution for the user will fail. With Cloudera Manager, the CDAP
+installation will create the ``cdap`` user on all nodes when it is distributed or
+activated on the cluster.
```

Comment:
fine for now, but would be nice to know which of "distributed" or "activated" it is... i dont know so would have to test.